src/status_im/subs/communities.cljs

```diff
@@ -359,3 +359,10 @@
         (map (fn [{sym :symbol image :image}]
                {sym image}))
         (into {}))))
+
+(re-frame/reg-sub
+ :communities/token-gated-not-a-member?
+ (fn [[_ community-id]]
+   [(re-frame/subscribe [:communities/community community-id])])
+ (fn [[{:keys [token-permissions joined]}] _]
+   (and (not joined) (seq token-permissions))))
```

Comment:
Pretty sure this doesn't return a boolean as `and` will return the first false/nil value in the form (left-to-right).
(`and` in Clojure works differently compared to other languages)

Maybe we should use `some?` instead of `seq` to check token permissions. WDYT?

Reply:
Thanks Mohamed, That sounds like a better idea.
Doing that now

Reply:
Done